src/main/java/dev/unnm3d/rediseconomy/currency/Currency.java

```diff
@@ -529,7 +529,7 @@ public EconomyResponse depositPlayer(@NotNull UUID playerUUID, @Nullable String
         return new EconomyResponse(amount, getBalance(playerUUID), EconomyResponse.ResponseType.SUCCESS, null);
     }
 
-    void updateAccountLocal(@NotNull UUID uuid, @Nullable String playerName, double balance) {
+    public void updateAccountLocal(@NotNull UUID uuid, @Nullable String playerName, double balance) {
```

Comment:
this shouldn't be used. there is already updateBulkAccountsCloudCache

Reply:
I used the same code
https://github.com/Emibergo02/RedisEconomy/blob/3338e61c6cf473a1e63d5525b0494580bf487c14/src/main/java/dev/unnm3d/rediseconomy/currency/CurrenciesManager.java#L117